scenes/game/local_game.gd

```diff
@@ -4390,6 +4394,10 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 					var card_range_max = card.definition['range_max']
 					# Some cards have mixed numbers and strings.
 					# Potentially evaluate? Currently only Phonon though and max==min if not in strike.
+					if card_range_min is String and card_range_min == "RANGE_TO_OPPONENT":
+						card_range_min = performing_player.distance_to_opponent()
+					if card_range_max is String and card_range_max == "RANGE_TO_OPPONENT":
+						card_range_max = performing_player.distance_to_opponent()
```

Comment:
Would it not be better to use get_card_stats? (not sure if there's anything else it matters for right now, but thinking ahead to Enchantress Homing Orb, specifically)